core/tests/protractor_utils/workflow.js

```diff
@@ -58,7 +60,21 @@ var createExplorationAndStartTutorial = function() {
   var creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage;
   creatorDashboardPage.get();
   // Wait for the dashboard to transition the creator into the editor page.
-  creatorDashboardPage.clickCreateActivityButton();
+  users.isAdmin().then(function(result) {
+    creatorDashboardPage.clickCreateActivityButton();
+    if (result) {
```

Comment:
This might be clearer if `result` were named `isAdmin` or similar

Reply:
Updated